app/views/works/edit.html.erb

```diff
@@ -0,0 +1,36 @@
+<main>
```

Comment:
This would be a good place to use a partial view to DRY up your code for the new and edit views.